save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/TestSuiteSourceCreationComponent.kt

```diff
@@ -112,6 +121,7 @@ private fun testSuiteSourceCreationComponent() = FC<TestSuiteSourceCreationProps
             responseHandler = ::responseHandlerWithValidation,
         )
         if (response.ok) {
+            fetchTestSuiteSource()
```

Comment:
Do we have an issue for better support of this fetch operation? Because it may b quite slow, and now we neither display status of fetching in progress nor indicate that it has started.

Reply:
I guess for now we don't. I think it might be nice to use something like web sockets for this kind of tasks.